web-client/src/pages/phone-number.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import CenteredCard from '../components/CenteredCard/CenteredCard';
+import GradientBackground from '../components/GradientBackground/GradientBackground';
+import PhoneNumber from '../components/PhoneNumber/PhoneNumber';
+
+const PhoneNumberPage: React.FC = () => (
+  <GradientBackground>
+    <CenteredCard>
+      <PhoneNumber />
+    </CenteredCard>
+  </GradientBackground>
+);
+
+export default PhoneNumberPage;
```

Comment:
This file should have the name `PhoneNumberPage.tsx` instead of `phone-number.tsx`